docs/components/Main/Main.js

```
@@ -11,22 +12,27 @@ import NavBar from './NavBar';
-                <div className='flex-shrink-0'>
+                <NavBarContext.Provider value={sendValues}>
+                <div className={`flex flex-col flex-1 h-screen ${darkMode ? 'rad-ui-dark-theme bg-black' : ''}`} data-accent-color="red">
+                    {/* Navbar start */}
                 <NavBar cookies={cookies} darkMode={darkMode} setDarkMode={setDarkMode} setCookie={setCookie} />
-                </div>
                 {/* Navbar end */}
-                <div className='flex flex-1 overflow-hidden'>
+                <>
                     {children}
-                </div>
+                </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)